hack/lib/mesh.bash

```diff
@@ -28,8 +30,45 @@ function deploy_servicemesh_operators {
 }
 
 function undeploy_servicemesh_operators {
+  logger.warn 'Teardown service mesh'
+  logger.info "Deleting service mesh CSVs"
+  if oc get subscription.operators.coreos.com servicemeshoperator -n openshift-operators >/dev/null 2>&1; then
+    CSV=$(oc get subscription.operators.coreos.com servicemeshoperator -n openshift-operators -o=custom-columns=CURRENT_CSV:.status.currentCSV --no-headers=true)
+    oc delete --ignore-not-found=true clusterserviceversions.operators.coreos.com $CSV -n openshift-operators
+  fi
+  if oc get subscription.operators.coreos.com kiali-ossm -n openshift-operators >/dev/null 2>&1; then
+    CSV=$(oc get subscription.operators.coreos.com kiali-ossm -n openshift-operators -o=custom-columns=CURRENT_CSV:.status.currentCSV --no-headers=true)
+    oc delete --ignore-not-found=true clusterserviceversions.operators.coreos.com $CSV -n openshift-operators
+  fi
+  if oc get subscription.operators.coreos.com jaeger-product -n openshift-operators >/dev/null 2>&1; then
+    CSV=$(oc get subscription.operators.coreos.com jaeger-product -n openshift-operators -o=custom-columns=CURRENT_CSV:.status.currentCSV --no-headers=true)
+    oc delete --ignore-not-found=true clusterserviceversions.operators.coreos.com $CSV -n openshift-operators
+  fi
+
+  logger.info "Deleting service mesh istio nodes"
+  oc delete --ignore-not-found=true daemonset.apps/istio-node -n openshift-operators
+  oc delete --ignore-not-found=true service/maistra-admission-controller -n openshift-operators
+
+  logger.info "Deleting service mesh webhooks and rbac resources"
+  oc delete --ignore-not-found=true validatingwebhookconfiguration openshift-operators.servicemesh-resources.maistra.io
```

Comment:
we followed the link how serverless stuff are uninstalled.
note: we need this because we are not deleting cluster every time before tests like x86 does. we ran cluster once and then one the same cluster we try multiple times tests